notebooks/COS/ViewData/ViewData.ipynb

```diff
@@ -126,29 +126,27 @@
     "\n",
```

Comment:
<p>Thank you for your valuable contribution following our previous discussions and style guidelines. I want to draw your attention to a minor issue: the table content headers 2.1 and 2.1.1 do not currently have any content linked. Your attention to this matter would be greatly appreciated.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>